cibyl/cli/main.py

```diff
@@ -42,36 +42,14 @@ def raw_parsing(arguments):
         if item == "--log-file":
             args["log_file"] = arguments[i + 2]
         elif item == "--log-mode":
-            args["log_mode"] = arguments[i+2]
+            args["log_mode"] = arguments[i + 2]
         elif item == "--debug":
             args["debug"] = True
         elif item in ('-p', '--plugin'):
-            if hasattr(args, 'plugins'):
-                args["plugins"].append(arguments[i + 2])
-            else:
-                args["plugins"] = [arguments[i + 2]]
+            args["plugin"] = arguments[i + 2]
```

Comment:
I think it might be worth to keep the ability to handle more than one plugin, even if we only have one use case for now

Reply:
Done